ui/articles-aem-renderer/src/main/java/org/cru/godtools/articles/aem/service/support/ArticleParser.java

```diff
@@ -31,14 +31,16 @@
     private static final String ORDER_FOLDER_TAG = "sling:OrderedFolder";
     private static final String PAGE_TAG = "cq:Page";
 
+    private static final String ARTICLE_BASE_URL = BASE_URL + "/content/experience-fragments/questions_about_god";
```

Comment:
we can't assume this is the base url, you should pass in whatever AEM import base URL you are actually parsing

Reply:
or base the url on that

Reply:
@frett is the "/content/experience-fragments/questions_about_god" always going to be the same?